use_joystick undeclared, compilation fails on Raspberry Pi

```
Making all in setup
make[3]: Entering directory `/home/pi/chocolate-doom/src/setup'
  CC     compatibility.o
  CC     display.o
  CC     joystick.o
joystick.c: In function ‘CalibrationEventCallback’:
joystick.c:508:17: error: ‘use_joystick’ undeclared (first use in this function)
joystick.c:508:17: note: each undeclared identifier is reported only once for each function it appears in
joystick.c: In function ‘NextCalibrateStage’:
joystick.c:421:1: warning: control reaches end of non-void function [-Wreturn-type]
make[3]: *** [joystick.o] Error 1
make[3]: Leaving directory `/home/pi/chocolate-doom/src/setup'
make[2]: *** [all-recursive] Error 1
make[2]: Leaving directory `/home/pi/chocolate-doom/src'
make[1]: *** [all-recursive] Error 1
make[1]: Leaving directory `/home/pi/chocolate-doom'
make: *** [all] Error 2
```

using Raspian with GCC 4.7.

--- a/src/setup/joystick.c
+++ b/src/setup/joystick.c
@@ -505,7 +505,7 @@ static int CalibrationEventCallback(SDL_Event *event, void *user_data)
             // Finished?
             if (calibrate_stage == CALIBRATE_CENTER)
             {
-                use_joystick = 1;
+                usejoystick = 1;
                 TXT_CloseWindow(calibration_window);
             }
 

